src/button-dropdown/__tests__/button-dropdown-items.test.tsx

```diff
@@ -284,6 +288,28 @@ const items: ButtonDropdownProps.Items = [
       expect(wrapper.findExpandableCategoryById('category1')!.getElement()).toHaveTextContent('category1');
     });
 
+    test('should render icons in expandable category headers', () => {
```

Comment:
This test suite has a `describe` block called `should render icons`. Would it make more sense to add this test in there and let its description just be `in expandable category headers`?